dbaas-cli/integtests/main.go

```diff
@@ -0,0 +1,54 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+)
+
+type TestEngine interface {
+	Run() error
+	CreateDB(rootPass string) (o string, err error)
+	ListDB() error
+	DescribeDB() error
+	ModifyDB() error
+	DeleteDB(bool) error
```

Comment:
Are you sure you need these methods in the interface since only Run() is used of the objects?